<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Templates</class>
 <widget class="QWidget" name="Templates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>870</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1021</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* В QSS-файле или строке */
QScrollArea {
    border: none;
    background: transparent;
}

/* Вертикальная полоса прокрутки */
QScrollBar:vertical {
    width: 10px;
    background: #2c2c2c;
}

QScrollBar::handle:vertical {
    background: #606060;
    border-radius: 5px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none;
}

/* Горизонтальная полоса прокрутки */
QScrollBar:horizontal {
    height: 10px;
    background: #2c2c2c;
}

QScrollBar::handle:horizontal {
    background: #606060;
    border-radius: 5px;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>870</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="nameTextEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>80</y>
     <width>341</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color: rgba(0, 85, 255, 55);
padding-left:10px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoName">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [UKWN]</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color: rgba(0, 85, 255, 55);
padding-left:10px</string>
   </property>
   <property name="text">
    <string> Найти</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>750</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40;
background-color: rgba(0, 85, 255, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 85, 255, 55);
border-radius: 26;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
